.github/workflows/ci.yaml

```
@@ -48,7 +48,7 @@ jobs:
 
       - id: setmatrix
         run: |
-          set stringified_matrix (tox -l | sed -e '/unit/d' -e '/get_urls/d' -e '/doc/d' -e '/lint/d' -e '/fips/d' | jo -a)
+          set stringified_matrix (tox -l | sed -e '/unit/d' -e '/get_urls/d' -e '/doc/d' -e '/lint/d' | jo -a)
```

Sorry for this suggestion, it's actually pointless, because we have no fips container for anything but 15.3 which we already add explicitly. This just adds 3 additional runs which skip all tests
```suggestion
          set stringified_matrix (tox -l | sed -e '/unit/d' -e '/get_urls/d' -e '/doc/d' -e '/lint/d' -e '/fips/d' | jo -a)
```

---

Done :heavy_check_mark: